ros2_batch_job/osx_batch/__init__.py

```diff
@@ -38,6 +38,12 @@ def pre(self):
             os.environ['LANG'] = 'en_US.UTF-8'
         if 'ROS_DOMAIN_ID' not in os.environ:
             os.environ['ROS_DOMAIN_ID'] = '111'
+        if 'OPENSSL_ROOT_DIR' not in os.environ:
+            import subprocess
+            brew_openssl_prefix = subprocess.run(
+                ['brew --prefix openssl'], shell=True, stdout=subprocess.PIPE)
+            if not brew_openssl_prefix.stdout is bytes():
+              os.environ['OPENSSL_ROOT_DIR'] = brew_openssl_prefix.stdout.decode().strip('\n')
```

Comment:
The environment variable is only assigned conditionally. What if the condition used for? What is supposed to happen when it is `False`?